Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "openssl-src"
-version = "110.0.7+1.1.0i"
+version = "110.0.8+1.1.1"
```

Comment:
Should this be `110.0.8` or `111.0.0`? Keeping it in the 110 series does mean we don't have to worry about maintaining 1.1.0 and 1.1.1 releases at the same time until 1.1.0 EOLs.

Reply:
Er oops yeah, let's switch to `111.0.0`